### PR TITLE
#14 Adds clean Jekyll blog file for duplication

### PR DESCRIPTION
**Pull request by trevormast**
Addresses issue #14 

Adds jekyll blog file to directory for duplication

Updates gemfile with jekyll dependency

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,6 +5,7 @@ gem 'sinatra'
 gem 'slim'
 
 group :development do
+  gem 'jekyll', '3.0.0'
```
Comment (amoose) [inline Gemfile]: I thought we decided to use [pessimistic versioning](https://robots.thoughtbot.com/rubys-pessimistic-operator)?

**Comment by amoose** [comment]
Looks great except for the version directive. Let's permit minor patches through by using pessimistic versioning.

**Comment by trevormast** [comment]
Good catch! Gemfile now updated to reflect pessimistic versioning for Jekyll dependency.

**Comment by amoose** [comment]
Wünderbar! :rocket:

**Pull request merged**